catroid/src/main/java/org/catrobat/catroid/io/asynctask/ProjectCopyTask.java

```diff
@@ -51,7 +53,8 @@ public ProjectCopyTask(Context context, ProjectCopyListener listener) {
 	}
 
 	@Override
-	protected Boolean doInBackground(String... strings) {
+	@VisibleForTesting
```

Comment:
the default for `otherwise` is `private`. I'm not sure if this task gets overloaded or not. If it does, then it should be `@VisibleForTesting(otherwise = VisibleForTesting.PROTECTED)`.